### ISSUE TITLE
AccessibleDOMNode : AccessibleNode is confusing

### ISSUE THREAD

**Issue by marcoscaceres** [a11y-api/a11y-api#39]
Can we please nave these things differently? It's quite confusing, particularly as all DOM nodes are by definition accessible.

**Comment by minorninth** [comment]
Agreed that AccessibleDOMNode is not as clear, we had a hard time coming up with consistent names

AccessibleNode is supposed to be a node in the accessibility tree, unrelated to a DOM node. There's precedent for this, for example AudioNode and all of its subclasses in the Web Audio API.

There are two types of AccessibleNodes, ones that were constructed by the browser and are associated with a DOM node, and ones that were constructed by the author.

Perhaps we don't need a subclass? We can just make the domNode property readonly...

**Comment by marcoscaceres** [comment]
> Perhaps we don't need a subclass? We can just make the domNode property readonly...

I think something like that could work. It would be similar to `.trusted` in DOM events.
